tests/conftest.py

```diff
@@ -240,6 +247,45 @@ def minimal_attribute_dict():
     }
 
 
+@pytest.fixture()
+def minimal_dataframe_dict():
+    """links transformers to minimal dataframes needed to successfully run transformer.
```

Comment:
N: Returns dict of transformer_name : minimal dataframe pairs.

Reply:
updated here https://github.com/lvgig/tubular/pull/253/commits/36cc5e1244eb45a7f358fdfb4f73659ecf7b0dae